.gitattributes

```diff
@@ -1 +1,2 @@
-*.a -text
```

Comment:
This is technically unrelated.  Something I did in my original version of this change.

I'm not sure what the old syntax is supposed to do?  Does "-text" work?   Anyway probably want to make this a separate change.

Reply:
I don't even know what this does, so I'll just revert it